### PR TITLE
[Onboarding] Add pre-push notification prompt

### PR DESCRIPTION
**Pull request by mennenia**
As per #1339.

Currently does not have a trigger on artist follow, I had questions around how to do that with the new Artist VC (cc @alloy @sarahscott). 

Also, analytics doesn't trigger at the moment and I'm not sure if I messed something up there.

Otherwise, working correctly on when the push notification shows at the moment. Go to settings works as well 👍

### CODE REVIEW

**Comment by sarahscott** [comment]
Looking good so far!

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">1 Error</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:no_entry_sign:</td>
      <td>No CHANGELOG changes made</td>
    </tr>
    
  </tbody>
</table>
  

  
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">2 Warnings</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td>PR is classed as Work in Progress</td>
    </tr>
        <tr>
      <td>:warning:</td>
      <td>Needs testing on a Phone if change is non-trivial</td>
    </tr>
    
  </tbody>
</table>
  

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

**Comment by mennenia** [comment]
Update: I've added the prompt in the artist follow areas, yet am still trying to work out the analytics. They seem to not be triggering.

**Comment by mennenia** [comment]
@sarahscott this is ready for review! :)

File: CHANGELOG.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ upcoming:
     details: Auctions Fixes?
 
   dev:
+    - Push notification on artist follow added - maxim
```
Comment (sarahscott) [inline CHANGELOG.yml]: this looks like valid YAML... @orta is the danger fail legit?

**Comment by orta** [comment]
Danger is true, it's not legit YAML:

```
[3] pry(main)> readme_data = YAML.load readme_yaml
Psych::SyntaxError: (<unknown>): did not find expected key while parsing a block mapping at line 1 column 1
from /Users/orta/.rvm/rubies/ruby-2.1.3/lib/ruby/2.1.0/psych.rb:370:in `parse'
```

**Comment by mennenia** [comment]
The YAML check was failing even before I added to the changelog.

Either way, this is going into a temp branch which I then need to merge with another temp branch. And I'll need to move up my changelog changes anyway, because they didn't get shipped in those releases, as this is all not in master yet.

I can deal with this separately on the proper onboarding branch, if that's okay with you all.

**Comment by sarahscott** [comment]
Merging!

**Comment by alloy** [comment]
🎉

**Pull request merged**